### ISSUE TITLE
conda-forge package

### ISSUE THREAD

**Issue by epassaro** [chianti-atomic/ChiantiPy#315]
Recently, I uploaded `ChiantiPy` to the `conda-forge` channel: https://anaconda.org/conda-forge/chiantipy. 

I think I should have to ask you before doing it, but it's already done. Currently versions available are 0.11.0 (latest) and 0.8.4 (required by  [`carsus`](https://github.com/tardis-sn/carsus)).

**Comment by kdere** [comment]
I am not sure how this will play out if ChiantiPy is updated
Ken

**Comment by wtbarnes** [comment]
Conda forge will automatically pick up each new release that gets uploaded to PyPI and open a PR against the chiantipy recipe repo. Releasing the new version on conda forge then is just a matter of merging that PR (provided the tests pass). E.g. the aiapy recipe repo: https://github.com/conda-forge/aiapy-feedstock

**Comment by epassaro** [comment]
Yes, this is an example of how the bot works: https://github.com/conda-forge/chiantipy-feedstock/pull/2. Human intervention is just required when you need to make changes to the package recipe (for example: adding new dependencies). 

People with write access to the feedstock repository are listed at the bottom of [the recipe](https://github.com/conda-forge/chiantipy-feedstock/blob/master/recipe/meta.yaml). Currently, I'm the only maintainer but that's not ideal. I can add more maintainers to the recipe.

I created the package because we want to release [`carsus`](https://github.com/tardis-sn/carsus) on `conda-forge` soon, and all the dependencies must exist on that channel.

**Comment by epassaro** [comment]
> Conda forge will automatically pick up each new release that gets uploaded to PyPI and open a PR against the chiantipy recipe repo. Releasing the new version on conda forge then is just a matter of merging that PR (provided the tests pass). E.g. the aiapy recipe repo: https://github.com/conda-forge/aiapy-feedstock

I enabled the automerge bot, so there's no need to do anything except for breaking changes (adding new dependencies, for example).
